gradle.properties

```diff
@@ -1,9 +1,27 @@
+# maven
 mavenUser=
 mavenPassword=
 mavenRepoUrl=http://nexus.4finance.net/content/repositories/Pipeline
+mavenProjectGroupId=com.ofg
+
+# Accurest
+accurestVersion=0.6.8
+# relative path to the Accurest Contract DSLs
+accurestRoot=repository/mappings
```

Comment:
eventually it's ending up as a file - this is only relative path. You can't access `Project` from props if I'm not mistaken.